### PR TITLE
CON-2595 Update will org be completing assessments page

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**
… completing assesments page

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationService.cs [hunk 1]
```diff
@@ -69,6 +118,8 @@ public async Task<CreateApplicationRequest> BuildCreateApplicationRequest(string
 
     public interface IApplicationService
```
Comment (cofaulco) [inline src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationService.cs:119]: Can we split the interface out into a separate file to be consistent with everywhere else?

**Review by cofaulco — COMMENTED** [review]
There don't appear to be any new/updated tests for this work as I would expect to see considering the changes.

File: src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationService.cs (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationService.cs]: Done

**Comment by ChrisJWoodcock** [comment]
> There don't appear to be any new/updated tests for this work as I would expect to see considering the changes.

I've added unit test for the pipelines count and starting a withdrawal application

**Review by cofaulco — APPROVED** [review]

**Pull request merged**